napari/_qt/_tests/test_qt_notifications.py

```diff
@@ -79,7 +78,6 @@ def test_notification_manager_via_gui(
         ]:
             notification_manager.records = []
             qtbot.mouseClick(btt, Qt.LeftButton)
```

Comment:
Does `qtbot.mouseClick` wait until the event has been processed?

Reply:
Here, the slot runs on the main thread. It then spins up a thread, but we wait on the main thread.